### PR TITLE
[4245] Set Loader top position to render in the middle of page

### LINKED ISSUE #4245: Mobile: Loader situated from the top when info about order is loading
**Description**:
Loader situated from the top when info about order is loading

**Expected behavior**:
Loader situated from in the middle of screen when info about order is loading

**Screenshots, Video, Logs**
image

**To Reproduce**:
1. Open FE
2. Login
3. Click My account > My orders 
4. Open any order

**Actual result**:
Loader situated from the top when info about order loading

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes #4245 

**Problem:**
* Loader situated from the top when info about order is loading

**In this PR:**
* Added optional mix prop on Loader to be able to style component position according to parent
* Set Loader's mix prop "MyAccountOrder-Loader" on MyAccountOrder.component 
* Set inset-block-start to 35vh on mobile

### CODE REVIEW

File: packages/scandipwa/src/component/Loader/Loader.component.js [hunk 1]
```diff
@@ -22,11 +22,13 @@ import './Loader.style';
  */
 export class Loader extends PureComponent {
     static propTypes = {
-        isLoading: PropTypes.bool
+        isLoading: PropTypes.bool,
+        mix: PropTypes.shape({ block: PropTypes.string, elem: PropTypes.string })
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/Loader/Loader.component.js:26]: There is mixType  for PropTypes. Check Types folder

Comment (pablobentivengo) [reply packages/scandipwa/src/component/Loader/Loader.component.js]: true.. Done!

**Comment by aleksandrakorolova** [comment]
Please, fix the same issue in Wishlist
https://drive.google.com/file/d/1VMU05Aqv-q4P06XpK95cfHUz4UXGDOGw/view?usp=sharing

**Comment by pablobentivengo** [comment]
Hey Aleksandra, I saw the same issue over the other tabs and fixed it. Let me know if it is Ok.

**Comment by aleksandrakorolova** [comment]
1. Fixed
2. Loader appears in the different place when opening my orders in the first time and when returning to list of orders from order page.
https://drive.google.com/file/d/1VWrgxv0yTsDPImJ_QS3kdkLL34DFJgi3/view?usp=sharing
Expected result:
Show loader in the middle of page (don't including header and account menu) all the time as in screenshot 
Screenshot 2022-02-11 at 09 25 51

**Comment by aleksandrakorolova** [comment]
2. Still, can reproduce
https://drive.google.com/file/d/1VnOZq3m27Y3vobFeWzCkaHX6l_EYnAWy/view?usp=sharing

File: packages/scandipwa/src/component/Loader/Loader.component.js [hunk 2]
```diff
@@ -12,6 +12,8 @@
 import PropTypes from 'prop-types';
 import { PureComponent } from 'react';
 
+import { MixType } from '../../type/Common.type';
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/Loader/Loader.component.js:15]: Use alias for import

File: packages/scandipwa/src/route/MyAccount/MyAccount.style.scss [hunk 3]
```diff
@@ -66,6 +66,14 @@
                     }
                 }
             }
+
+            &_my-wishlist, &_my-orders {
+                .Loader-Scale {
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/route/MyAccount/MyAccount.style.scss:71]: .Loader {
   &-Scale
   .....

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/Loader/Loader.component.js (hunk 2 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/Loader/Loader.component.js:15]: Done

File: packages/scandipwa/src/route/MyAccount/MyAccount.style.scss (hunk 3 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/route/MyAccount/MyAccount.style.scss:71]: Done

File: packages/scandipwa/src/route/MyAccount/MyAccount.style.scss [hunk 4]
```diff
@@ -66,6 +66,16 @@
                     }
                 }
             }
+
+            &_my-wishlist, &_my-orders {
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/route/MyAccount/MyAccount.style.scss:70]: Move new selector to new line

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**